activesupport/lib/active_support/core_ext/module/attribute_accessors_per_thread.rb

```diff
@@ -136,9 +140,9 @@ def #{sym}=(obj)
   #
   #   Current.new.user = "DHH"  # => NoMethodError
   #   Current.new.user          # => NoMethodError
-  def thread_mattr_accessor(*syms, instance_reader: true, instance_writer: true, instance_accessor: true)
-    thread_mattr_reader(*syms, instance_reader: instance_reader, instance_accessor: instance_accessor)
-    thread_mattr_writer(*syms, instance_writer: instance_writer, instance_accessor: instance_accessor)
+  def thread_mattr_accessor(*syms, instance_reader: true, instance_writer: true, instance_accessor: true, default: nil)
+    thread_mattr_reader(*syms, instance_reader: instance_reader, instance_accessor: instance_accessor, default: default)
```

Comment:
Do we need to set the default value on the reader since the writter will also set the value?